nestkernel/source_table.h

```diff
@@ -64,17 +64,17 @@ class SourceTable
   /**
    * 3D structure storing gids of presynaptic neurons.
    */
-  std::vector< std::vector< std::vector< Source >* >* > sources_;
+  std::vector< std::vector< std::vector< Source > > > sources_;
```

Comment:
the reasoning behind using this structure for the vector was to allow the allocator to create the actual objects
1. independent of each other in the memory space of a specific thread (1st dimension) to improve access speed
2. independent of each other according to synid (2nd dimension) to allow vectors for different syn ids to be distributed in memory to improve freeing of memory when deleting the innermost vectors

are both these conditions still fulfilled if we do not use pointers here?